docker-compose.yml

```diff
@@ -26,6 +26,8 @@ services:
       - DATABASE_URL=postgres://app:secret@postgres/app_development
       - DEV_HOST=${DEV_HOST:-localhost} # Set if you want to use a different hostname
       - ALLOWED_REQUEST_ORIGINS=${ALLOWED_REQUEST_ORIGINS}
+      - DEFAULT_OPENAI_KEY={DEFAULT_OPENAI_KEY}
+      - DEFAULT_ANTHROPIC_KEY={DEFAULT_ANTHROPIC_KEY}
```

Comment:
I believe this needs $ it front of the curlies. At the last the one above it has that

Reply:
While you're adding these, can you do me a favor and make sure every Setting ENV is listed here? We're missing a few.

Also, add a comment such as:

```
  environment: -- Be sure to add this environment variable to options.yml
```

And do the reverse too, in the `options.yml` file on the settings line do:

```
  settings: -- Be sure to add these ENV to docker-compose.yml
```

This project is getting nuanced enough it's even hard for me to remember all the subtle connections so sprinkling these hints will help future me (and you) :)